nav2_theta_star_planner/README.md

```diff
@@ -68,20 +67,25 @@ planner_server:
 ```
 Do note that the `global_costmap`'s `inflation_layer` values recommended for the above values of the parameter are - `cost_scaling_factor:1.0`, `inflation_radius: 5.5`
 
-## Usage Notes
+##Usage Notes
 
-### Tuning the Parameters
-Before starting off, do note that the costmap_cost(curr,neigh) component after being operated (before being multiplied to its parameter and being substituted in g(node)) varies from 0 to 1. Keep this in mind while tuning.
+###Tuning the Parameters
+Before starting off, do note that the costmap_cost(curr,neigh) component after being operated (before being multiplied to its parameter and being substituted in g(init_rclcpp)) varies from 0 to 1. Keep this in mind while tuning.
```

Comment:
Why are all the "node" changed to "init_rclcpp" in the readme?

Reply:
Fixed it!